tests/unittests/test_dftb_feeds.py

```diff
@@ -552,3 +666,18 @@ def test_hubbardfeed_batch(device, skf_file):
     check_2 = torch.allclose(predicted, reference)
     assert check_2, 'Predicted hubbard value errors exceed allowed tolerance'
 
+
+if __name__ == '__main__':
```

Comment:
Hi Adam, seems these codes are only for debugging :)

Reply:
Removed debugging code block.